modules/ROOT/pages/manage-members.adoc

```diff
@@ -25,13 +25,16 @@ Consumer developers follow these steps to self-register:
 . After a registration request is sent by the member, you will receive a notification email. To review and
 approve the member registration, navigate to the ACM Control Panel.
 . Select *Manage Members*. This section provides the status of all consumer developer registration requests.
-. Select *Approve* to activate the member and trigger a welcome email.
-After you approve the registration request, a welcome email is sent to the consumer developer.
+. To activate the member and trigger a welcome email, select *Approve* and confirm the change.
+. To reject a registration request, select *Reject* and confirm the change.
+. After you approve a registration request, a welcome email is sent to the consumer developer.
 . To complete registration, the member must click an email verification link in the email and specify a new
 password. The consumer developer can then log in to your community.
 When the member's status is *Active*, additional options become available to the developer, such as *Forums*
 and *My Applications*.
-. (Optional) To deactivate a member, disable the *Active* setting for that member.
+. (Optional) To deactivate a member, disable the *Active* setting for that member and confirm the change.
+
+Select the options menu to choose how many registration requests to show on each page, and to *Show rejected users* or *Hide rejected users*.
```

Comment:
Select the options menu to specify the number of registration requests to display on each page, and to *Show rejected users* or *Hide rejected users*.

Reply:
Much better! Fixed.